example/bookstore/v1/bookstore.proto

```diff
@@ -130,7 +130,7 @@ message GetAuthorResponse {
 // A shelf resource.
 message Shelf {
   // A unique shelf id.
-  int64 id = 1;
+  string id = 1;
```

Comment:
hmm, this is a rough.... change? to not support ints in paths.

Reply:
Only when using nested objects in route paths (`/api/{obj.parent_id}/blah/{obj.id}`), which was broken previously. The behavior with normal params (`/api/{parent_id}/blah/{obj_id}`) is unchanged and works fine with non-strings.

Reply:
Oh and the string limitation is due to https://github.com/ductone/protoc-gen-apigw/blob/0d29824d369c4b77554a1a0200f7fd9d59a565d7/internal/apigw/templates/protopack_accessor.tmpl#L5 which I didn't change.